app_dart/test/request_handlers/get_branches_test.dart

```diff
@@ -162,5 +217,41 @@ void main() {
       expect((result.single)['branch']['branch'], 'flutter-branch-created-now');
       expect((result.single)['id'].runtimeType, String);
     });
+
+    test('should always retrieve release branches', () async {
+      expect(db.values.values.whereType<Branch>().length, 1);
+
+      const String id = 'flutter/flutter/$betaBranchName';
+      final int lastActivity = DateTime.tryParse("2019-05-15T15:20:56Z")!.millisecondsSinceEpoch;
+      final Key<String> branchKey = db.emptyKey.append<String>(Branch, id: id);
+      final Branch currentBranch = Branch(
+        key: branchKey,
+        lastActivity: lastActivity,
+      );
+      db.values[currentBranch.key] = currentBranch;
+
+      const String stableId = 'flutter/flutter/$stableBranchName';
+      final Key<String> stableBranchKey = db.emptyKey.append<String>(Branch, id: stableId);
+      final Branch stableBranch = Branch(
+        key: stableBranchKey,
+        lastActivity: lastActivity,
+      );
+      db.values[stableBranch.key] = stableBranch;
+
+      expect(db.values.values.whereType<Branch>().length, 3);
+
+      final List<dynamic> result = (await decodeHandlerBody())!;
+      final List<dynamic> expected = [
+        {
+          'id': 'flutter/flutter/flutter-beta',
+          'branch': {'branch': 'flutter-beta', 'repository': 'flutter/flutter'},
```

Comment:
Should branch.branch be `beta`?

Reply:
umm the branch.branch will be the actual name of the branch. e.g. a beta branch would have the name `flutter-3.10-candidate.1`